hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/pom.xml

```diff
@@ -208,6 +208,12 @@
       <artifactId>jersey-media-json-jettison</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.glassfish.jaxb</groupId>
+      <artifactId>jaxb-runtime</artifactId>
+      <version>${jaxb.version}</version>
+      <scope>test</scope>
```

Comment:
This is needed by Jetty. (at least)
This should be compile scope, as javax.xml.bind is directly used in the main code.

Reply:
I was not sure this needed in every case for the Jetty.
But in that case if i see well we will need this every where where we are using the HttpServer2.
So if i see well this will be needed in HDFS NN also.

In that case shall we add this to the just to the `hadoop-common` project and let others use it transitively?

Reply:
One other issue is the binary assembly

We also need to make sure that jaxb-impl is included there.